docs/tags/common-schema.md

```diff
@@ -41,14 +41,49 @@ The tags below are applicable to all metrics/logs types:
 
 ## Operating System Tags
 
-Below are the OS related tags sent as part of OS metrics/logs
+Below are the OS related tags sent as part of OS metrics/logs. They are collected from the host the [Sematext Agent](../agents/sematext-agent) is running on. They're mapped to the `os.host` tag.
 
 | Tag Name  | Description  | Synonymous Tags
 |:--|:--|:--
-| os.disk | Human readable name of the block device | 
-| os.disk.mountpoint | Mount point for the disk in the file system |
-| os.disk.fs.type | Type of the file system associated with the device |
-| os.network | Name of the network interface |
+| System Disk | os.disk | Human readable name of the block device | 
+| System Disk Mountpoint| os.disk.mountpoint | Mount point for the disk in the file system |
+| Filesystem Type | os.disk.fs.type | Type of the file system associated with the device |
+| System Network Interface | os.network | Name of the network interface |
+| System UUID | os.uuid | Unique ID based on SMBIOS specification |
+| OS Distribution Name | os.distro.name | Distribution name of the OS. e.g. `ubuntu` |
+| OS Distribution Version | os.distro.version | Version of the OS. e.g. `16.04` |
+| Kernel Version | os.kernel | Version of the Kernel. e.g. `4.4.0-130-generic` |
+| JVM Version | jvm.version | Version of JVM, if available in `PATH` |
+| Virtualization | virtualization | Virtualization Type. Possible values are `BareMetal`, `VM`, `Container` |
+| Container Labels | - | All user-defined container labels. These tag are mapped to `os.host` and `container.id` |
+
+## Cloud Tags
+
+The cloud metadata from AWS, Azure and GCE instances are collected as tags. They're mapped to the `os.host` tag.
```

Comment:
Again "mapped to" - one could imagine there is some sort of "mapping" in case of tag aliases, but since we don't want to differentiate between the tag types anymore, I think we shouldn't mention any relation to os.host.